### ISSUE TITLE
Dashboard: Metrics: Need an option to split a metric based on a dimension

### ISSUE THREAD

**Issue by samsp-msft** [dotnet/aspire#4906]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

The OTel semantic conventions tend to use a single metric to represent variations on operations, and then use dimensions to split the values. For example [gen_ai.client.token.usage](https://github.com/open-telemetry/semantic-conventions/blob/f79fe6b61438bfa0804df9c254c402fd0d71f42a/docs/gen-ai/gen-ai-metrics.md#metric-gen_aiclienttokenusage) is a metric for LLMs to represent the number of tokens used in a LLM call. The same metric is used for both input and output, but has a dimension `gen_ai.token.type` to identify which type it is.

Using the same graph line for these two usages is wrong - the values could be orders of magnitude different.

### Describe the solution you'd like

There should be an option to split the graph based on a dimension. As part of the dimension list, there should be a checkbox for each dimension as to whether to split the results - drawing independent lines for each value of that dimension.

This should probably only be available for a single dimension at a time - doing it for multiple will result in an explosion of lines and it will be hard to follow.

**Comment by joperezr** [comment]
@leslierichardson95 can you make sure this is in the right milestone?
